src/main/java/com/opal/App.java

```diff
@@ -8,13 +8,19 @@
 {
     public static void main( String[] args )
     {
+        System.out.println( "Hello World!" );
+
+        System.out.println( "Print for users/Rafael!" );
+
         System.out.println( "Hello World!" + sergey());
         System.out.println( "Hello World!" );
         System.out.println("Hello world dudi");
         //one one one
   
+
         //ira commit
         //Commited from Vitaly to feature
+        //Hello World
     }
 
     public static String sergey(){return new String(" Sergey");}
```

Comment:
Our static analysis engine detects that your pull request includes a change that method invokes inefficient new String(String) constructor. Using the `java.lang.String(String)` constructor wastes memory because the object so constructed will be functionally indistinguishable from the `String` passed as a parameter.  Just use the argument `String` directly.